### PR TITLE
Added LICENSE

### PR DESCRIPTION
**Pull request by Preston159**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+possible use to the public, the best way to achieve this is to make it
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+state the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    <one line to give the program's name and a brief idea of what it does.>
+    Copyright (C) <year>  <name of author>
```
Comment (Vilsol) [inline LICENSE:635]: Missing information

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,674 @@
...
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+Also add information on how to contact you by electronic and paper mail.
+
+  If the program does terminal interaction, make it output a short
+notice like this when it starts in an interactive mode:
+
+    <program>  Copyright (C) <year>  <name of author>
```
Comment (Vilsol) [inline LICENSE:655]: Missing information

**Review by Vilsol — CHANGES_REQUESTED** [review]
There is a bigger issue than just adding a file called `LICENSE` to the repository. We must first request every previous contributor to agree to this license before we can enforce it. Have you done this?

**Comment by Preston159** [comment]
Firstly, those lines in the license file are not meant to be modified (and, in fact, cannot be). They exist as examples of what should be displayed _in_ your program.

```
Copyright (C) 2007 Free Software Foundation, Inc. <https://fsf.org/>
 Everyone is permitted to copy and distribute verbatim copies
 of this license document, but changing it is not allowed.
```

Regarding the requirement that contributors agree, I think we'd only need 3 of the existing contributors to agree:

- @Vilsol 
- @Panakotta00 
- @budak7273 

As far as I can tell, these are the only three people who have made contributions outside of the documentation/website, which already states at the bottom that it is licensed under GPLv3; this is why I chose the same license. If I'm incorrect about any of this, please let me know.

**Comment by Vilsol** [comment]
When you are referencing `documentation/website`, what are you referring to specifically?

Also, we would have to get confirmation from all of [these](https://github.com/satisfactorymodding/Documentation/graphs/contributors) contributors, as a root license would apply to everything in the repository.

**Comment by Preston159** [comment]
In the footer of https://docs.ficsit.app/ it states

> Except where otherwise noted, docs.ficsit.app is licensed under the GNU General Public License v3.0 (GPLv3).

Unless there exists content served on the website that states otherwise, I take this to mean everyone who has contributed in the `modules` and `supplemental-ui` folder has already agreed to the GPLv3 license.

**Comment by aaomidi** [comment]
> Unless there exists content served on the website that states otherwise, I take this to mean everyone who has contributed in the modules and supplemental-ui folder has already agreed to the GPLv3 license.

https://github.com/satisfactorymodding/Documentation/tree/9ed77e86b9889f1b5f1b938726ea391164604c7d

IANAL. This is where that language was introduced, anything before that point will need permissions from contributors most likely.

**Comment by Feyko** [comment]
Closing this because of [this](https://github.com/satisfactorymodding/Documentation/pull/96#issuecomment-989152231) comment, the following mini-thread about the current effective License and inactivity

**Pull request closed without merging**